server/assets/tests/specs/invoiceItemSpec.js

```diff
@@ -8,19 +8,9 @@ var superagent = require('superagent');
 describe('Invoice item tests', function() {
 
   var URL = 'https://localhost:4443';
-  // var invoiceObj = {
-  //   title: 'test invoice',
-  //   description: 'invoice',
-  //   poNumber: 123454,
-  //   invoiceNumber: 1111,
-  //   taxApplied: true,
-  //   feeApplied: true,
-  //   status: 'entered',
-  //   invoiceItems: []
-  // };
-  var currentInvoiceId = "558317feee627fa68f5aab21";
+  var currentInvoiceId = '558317feee627fa68f5aab21';
   var invoiceItemObj = {
-    invoiceId: "558317feee627fa68f5aab21",
+    invoiceId: '558317feee627fa68f5aab21',
```

Comment:
We should be setting the invoice id based on the invoice that we will be creating for the test. This hard coding of the invoiceId will not allow for a robust test.